D_controllers.md

```diff
@@ -76,21 +87,27 @@ The data world is your oyster, but we won't be covering these options in these g
 
 There are times when we need to communicate with users during the course of an action. Maybe there was an error updating a model. Maybe we just want to welcome them back to the application. For this, we have flash messages.
 
-The `Phoenix.Controller` module provides the `put_flash/3` and `get_flash/2` functions to help us set and retrieve flash messages as a key value pair. Let's set two flash messages in our `HelloPhoenix.PageController` to try this out.
+The `Phoenix.Controller` module provides the `put_flash/3` and `get_flash/2` functions to help us set and retrieve flash messages as a key value pair. Let's set two flash messages in our `HelloPhoenix.PageController` in `web/controllers/page_controller.ex` to try this out.
```

Comment:
Reverted.

Still I'd like to suggest using a more consistent manner of indicating which file the code snippets come from as it is sometimes confusing for newbies to figure this out.

Either a commented out file path at the top of each code snippet, or a link to the github file perhaps.